app/controllers/api/profiles/testimonials_controller.rb

```diff
@@ -3,5 +3,44 @@ class TestimonialsController < BaseController
     def index
       render json: AssembleProfileTestimonialsList.(@user)
     end
+
+    def hiring
+      testimonials = @user.mentor_testimonials.published.joins(solution: { exercise: :track })
```

Comment:
Do we want to fix the user? This currently depends on the user sending an authenticated request.

Reply:
It's actually the user who's profile you're looking at. I think it's good for giving some variety.

Reply:
Okay, that's fine. As long as it's a deliberate choice :)

Reply:
Thanks!